_docs/howto/configure/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Configure Forseti Security Features
+order: 101
```

Comment:
(seems github is having issues and my comment was lost)

All articles under `/configure/` will need the sam leading digit.